sdk/ml/azure-ai-ml/azure/ai/ml/operations/_evaluator_operations.py

```diff
@@ -0,0 +1,772 @@
+        """
+
+        self.get(name=name, version=version)
+
+        #  Get workspace info to get workspace GUID
+        workspace = self._service_client.workspaces.get(
+            resource_group_name=self._resource_group_name,
+            workspace_name=self._workspace_name,
+        )
+        workspace_guid = workspace.workspace_id
+        workspace_location = workspace.location
+
+        # Get model asset ID
+        asset_id = ASSET_ID_FORMAT.format(
+            workspace_location,
+            workspace_guid,
+            AzureMLResourceType.MODEL,
+            name,
+            version,
+        )
+
+        model_ref = WorkspaceAssetReference(
+            name=share_with_name if share_with_name else name,
+            version=share_with_version if share_with_version else version,
+            asset_id=asset_id,
+        )
+
+        with self._set_registry_client(registry_name):
+            return self.create_or_update(model_ref)
+
+    def _get_latest_version(self, name: str) -> Model:
+        """Returns the latest version of the asset with the given name.
+
+        Latest is defined as the most recently created, not the most recently updated.
+        """
+        result = _get_latest(
+            name,
+            self._model_versions_operation,
+            self._resource_group_name,
+            self._workspace_name,
+            self._registry_name,
+        )
+        return Model._from_rest_object(result)
+
+    @contextmanager
+    def _set_registry_client(self, registry_name: str) -> Generator:
+        """Sets the registry client for the model operations.
+
+        :param registry_name: Name of the registry.
+        :type registry_name: str
+        """
+        rg_ = self._operation_scope._resource_group_name
+        sub_ = self._operation_scope._subscription_id
+        registry_ = self._operation_scope.registry_name
+        client_ = self._service_client
+        model_versions_operation_ = self._model_versions_operation
+
+        try:
+            _client, _rg, _sub = get_registry_client(self._service_client._config.credential, registry_name)
+            self._operation_scope.registry_name = registry_name
+            self._operation_scope._resource_group_name = _rg
+            self._operation_scope._subscription_id = _sub
+            self._service_client = _client
+            self._model_versions_operation = _client.model_versions
+            yield
+        finally:
+            self._operation_scope.registry_name = registry_
+            self._operation_scope._resource_group_name = rg_
+            self._operation_scope._subscription_id = sub_
+            self._service_client = client_
+            self._model_versions_operation = model_versions_operation_
+
+    @experimental
+    @monitor_with_activity(ops_logger, "Model.Package", ActivityType.PUBLICAPI)
+    def package(self, name: str, version: str, package_request: ModelPackage, **kwargs: Any) -> Environment:
```

Comment:
Not sure what's the package API used for

Reply:
Removed. This API needed to create a docker container to run the model and does not make sense in the context of evaluator.